frontend/reports/reports.components.tsx

```diff
@@ -80,6 +82,11 @@ export default function Reports(props: ReportsProps) {
               title="Ages and Genders"
             />
             <AgesAndGendersResults path="ages-and-genders/results" />
+            <ClientZipcodeParams
+              path="client-zipcode"
```

Comment:
The way that the reports are set up requires that the frontend route match the backend API's path. So this line needs to match https://github.com/JustUtahCoders/comunidades-unidas-internal/pull/689/files#diff-ae17fbe5ac7201c5776b7888b1827149906ec26ddc054a936017aa1f5825886cR7

```suggestion
              path="client-zipcodes"
```

Reply:
I have fixed this. Got tripped up with the singular vs plural